api/integral-time-weight.md

```diff
@@ -0,0 +1,73 @@
+---
+api_name: integral()
+excerpt: Calculate the time-weighted integral of values in a `TimeWeightSummary`
+topics: [hyperfunctions]
+keywords: [average, time-weighted, hyperfunctions, toolkit]
+api:
+  license: community
+  type: function
+  experimental: true
+  toolkit: true
+hyperfunction:
+  family: time-weighted averages
+  type: accessor
+  aggregates:
+    - time_weight()
+# fields below will be deprecated
+api_category: hyperfunction
+api_experimental: true
+toolkit: true
+hyperfunction_family: 'time-weighted averages'
+hyperfunction_subfamily: 'time-weighted averages'
+hyperfunction_type: accessor
+---
+
+# integral() <tag type="toolkit">Toolkit</tag><tag type="experimental-toolkit">Experimental</tag>
+
+```SQL
+integral(
+    tws TimeWeightSummary,
+    unit TEXT
+) RETURNS DOUBLE PRECISION
+```
+
+A function to compute a time-weighted integral from a `TimeWeightSummary`.
+
+This function is similar to [`average`][hyperfunctions-average] but doesn't divide by the length of time being integrated.
+
+*   For more information about time-weighted average functions, see the
+    [hyperfunctions documentation][hyperfunctions-time-weight-average].
+*   For more information about statistical aggregate functions, see the
+    [hyperfunctions documentation][hyperfunctions-stats-agg].
```

Comment:
```suggestion
```

Any particular reason you want to include this link? My view is that it's a little confusing because it's unclear what the connection is to stats aggs, but I'm happy to be convinced otherwise

Reply:
I just copied it from the average docs. I've removed the link from all of the time weight functions that had it.